pkg/kubelet/certificate/kubelet.go

```diff
@@ -90,13 +92,28 @@ func NewKubeletServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
 	)
-	legacyregistry.MustRegister(certificateRotationAge)
+	registerMetricsOnce.Do(func() {
```

Comment:
hmm, if newCertificateManagerConfig is called more than once, the returned configs will have metrics that aren't actually registered or reporting anywhere

before, MustRegister would panic on this... now, we just silently return not-working configs

if we're going to let this be called multiple times, I think the certificateRenewFailure / certificateRotationAge have to become shared package vars that lazily init / register once on first call

Reply:
> the returned configs will have metrics that aren't actually registered or reporting anywhere

metrics are global and you can only register once, so if is called multiple times the metrics will still apply

> before, MustRegister would panic on this... now, we just silently return not-working configs

why will be not-working? 

> if we're going to let this be called multiple times, I think the certificateRenewFailure / certificateRotationAge have to become shared package vars that lazily init / register once on first call

I started with that but the diff was larger( thinking on backports)

Reply:
hmm, I didn't realize there is another certficiate.Config below 🤔